soundfile.py

```diff
@@ -563,7 +702,7 @@ def default_subtype(format):
 
     """
     _check_format(format)
-    return _default_subtypes.get(format.upper())
+    return _default_subtypes.get(cast(Format, format.upper()))
```

Comment:
why was a cast added here?

Reply:
Fixed in https://github.com/bastibe/python-soundfile/pull/462/commits/a8a77c1e8155dc029f53aad1d69a7a501a60a671